src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -126,5 +132,21 @@ public boolean onShoot(ProjectileHitEvent e){
            }
            return false;
        }
+       public boolean isArrow(ItemStack item){
+        if(item.getType()== Material.TIPPED_ARROW|| item.getType() == Material.ARROW || item.getType()==Material.SPECTRAL_ARROW){
```

Comment:
you can make this simplier by doing just return (item.getType()== Material.TIPPED_ARROW|| item.getType() == Material.ARROW || item.getType()==Material.SPECTRAL_ARROW)

Reply:
done